scripts/nlp_api_master.py

```diff
@@ -69,82 +245,115 @@
+    "children": [
+        {
+            "path": "mlm_mask_generator",
+            "title": "MLMMaskGenerator",
+            "generate": ["keras_nlp.layers.MLMMaskGenerator"],
+        },
+        {
+            "path": "mlm_head",
+            "title": "MLMHead",
+            "generate": ["keras_nlp.layers.MLMHead"],
+        },
+        {
+            "path": "start_end_packer",
+            "title": "StartEndPacker",
+            "generate": ["keras_nlp.layers.StartEndPacker"],
+        },
+        {
+            "path": "multi_segment_packer",
+            "title": "MultiSegmentPacker",
+            "generate": ["keras_nlp.layers.MultiSegmentPacker"],
+        },
+        {
+            "path": "random_swap",
+            "title": "RandomSwap",
+            "generate": ["keras_nlp.layers.RandomSwap"],
+        },
+        {
+            "path": "random_deletion",
+            "title": "RandomDeletion",
+            "generate": ["keras_nlp.layers.RandomDeletion"],
+        },
+    ],
+}
+
+MODELING_LAYERS_MASTER = {
+    "path": "modeling_layers/",
+    "title": "Modeling Layers",
     "toc": True,
     "children": [
         {
             "path": "transformer_encoder",
-            "title": "TransformerEncoder layer",
+            "title": "TransformerEncoder",
             "generate": [
                 "keras_nlp.layers.TransformerEncoder",
                 "keras_nlp.layers.TransformerEncoder.call",
             ],
         },
         {
             "path": "transformer_decoder",
-            "title": "TransformerDecoder layer",
+            "title": "TransformerDecoder",
             "generate": [
                 "keras_nlp.layers.TransformerDecoder",
                 "keras_nlp.layers.TransformerDecoder.call",
             ],
         },
         {
             "path": "fnet_encoder",
-            "title": "FNetEncoder layer",
+            "title": "FNetEncoder",
             "generate": ["keras_nlp.layers.FNetEncoder"],
         },
         {
             "path": "position_embedding",
-            "title": "PositionEmbedding layer",
+            "title": "PositionEmbedding",
             "generate": ["keras_nlp.layers.PositionEmbedding"],
         },
         {
             "path": "sine_position_encoding",
-            "title": "SinePositionEncoding layer",
+            "title": "SinePositionEncoding",
             "generate": ["keras_nlp.layers.SinePositionEncoding"],
         },
         {
             "path": "token_and_position_embedding",
-            "title": "TokenAndPositionEmbedding layer",
+            "title": "TokenAndPositionEmbedding",
             "generate": ["keras_nlp.layers.TokenAndPositionEmbedding"],
         },
-        {
-            "path": "mlm_mask_generator",
-            "title": "MLMMaskGenerator layer",
-            "generate": ["keras_nlp.layers.MLMMaskGenerator"],
-        },
         {
             "path": "mlm_head",
-            "title": "MLMHead layer",
+            "title": "MLMHead",
```

Comment:
We should be consistent with core Keras docs, which currently have "layer" in the page title for all layers, e.g. https://keras.io/api/layers/normalization_layers/batch_normalization/